stack-qa/recipes/deploy-sinopia-nginx.rb

```diff
@@ -2,11 +2,11 @@
 
   next unless allow_deploy(application, 'sinopia', 'npm-proxy')
 
-  if node.fetch('sinopia', {})['listen'].nil?
-    sinopia_listener = '127.0.0.1:4873'
-  else
-    sinopia_listener = node['sinopia']['listen']
-  end
+  sinopia_listener = if node.fetch('sinopia', {})['listen'].nil?
```

Comment:
Different. I personally consider it to be nicer as in better readable, but not sure. See RfC in the Ops flow